### PR TITLE
Update API env names

### PR DESCRIPTION
**Pull request by ofalvai**
### Checklist

- [ ] I've read and followed the [Contribution Guidelines](https://github.com/bitrise-steplib/.github/blob/main/CONTRIBUTING.md)
- [ ] `step.yml` and `README.md` is updated with the changes (if needed)

### Version

Requires a *MAJOR/MINOR/PATCH* [version update](https://semver.org/)

### Context

Finalizing API env names exposed by the VM.

### CODE REVIEW

File: e2e/bitrise.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ workflows:
         - content: |-
             set -ex
             npm install
-    - git::https://github.com/bitrise-steplib/bitrise-step-save-cache.git@main:
```
Comment (imrekel) [inline e2e/bitrise.yml:46]: I guess this should remain `main`

File: e2e/bitrise.yml [hunk 2]
```diff
@@ -99,7 +99,7 @@ workflows:
         inputs:
         - module: app
         - variant: debug
-    - git::https://github.com/bitrise-steplib/bitrise-step-save-cache.git@main:
```
Comment (imrekel) [inline e2e/bitrise.yml:102]: same here

File: e2e/bitrise.yml [hunk 3]
```diff
@@ -175,4 +175,5 @@ workflows:
 
             auth_token=$(echo $json_response | jq -r .access_token)
 
-            envman add --key BITRISEIO_CACHE_SERVICE_ACCESS_TOKEN --value $auth_token --sensitive
+            envman add --key BITRISEIO_ABCS_API_URL --value $BITRISEIO_CACHE_SERVICE_URL --sensitive
```
Comment (imrekel) [inline e2e/bitrise.yml:178]: just a note: this is added as non-sensitive env (not a secret) in the normal build env

**Review by imrekel — CHANGES_REQUESTED** [review]

File: e2e/bitrise.yml (hunk 1 above)
Comment (ofalvai) [reply e2e/bitrise.yml]: Yes, now it's working with `main`. I reverted these changes.

File: e2e/bitrise.yml (hunk 3 above)
Comment (ofalvai) [reply e2e/bitrise.yml]: Good to know, I changed it to non-sensitive.

**Review by imrekel — APPROVED** [review]

**Pull request merged**